### PR TITLE
Update footer June 22

### PR DESCRIPTION
**Pull request by ahmadabdolsaheb**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: build/universal/trending/english.yaml [hunk 1]
```diff
@@ -1,60 +1,62 @@
...
+	article25link: 'https://www.freecodecamp.org/news/default-constructor-in-java/',
+	article26title: 'Stuck Win 10 Hard Drive',
+	article26link: 'https://www.freecodecamp.org/news/scanning-and-repairing-drive-how-to-fix-stuck-windows-10-pc-hard-drive/',
+	article27title: 'Color Codes for Grey Palette',
+	article27link: 'https://www.freecodecamp.org/news/color-shades-hex-code-and-rgb-for-gray-color-palette/',
+	article28title: 'Binary Search Tree Traversal',
+	article28link: 'https://www.freecodecamp.org/news/binary-search-tree-traversal-inorder-preorder-post-order-for-bst/',
+	article29title: 'RTC Connecting Discord Fix',
+	article29link: 'https://www.freecodecamp.org/news/rtc-connecting-discord-how-to-fix-the-server-error/'
+  }
```
Comment (ahmadabdolsaheb) [inline build/universal/trending/english.yaml:62]:
```suggestion
{
article0link: "https://www.freecodecamp.org/news/discord-wont-open-on-my-pc-solved-in-windows-10/"
article0title: "Discord Won't Open"
article10link: "https://www.freecodecamp.org/news/c-vs-cpp-whats-the-difference/"
article10title: "C vs C++"
article11link: "https://www.freecodecamp.org/news/python-json-how-to-convert-a-string-to-json/"
article11title: "Python JSON"
article12link: "https://www.freecodecamp.org/news/what-is-a-pc-computer-definition-and-computer-basics-for-beginners/"
article12title: "What is a PC?"
article13link: "https://www.freecodecamp.org/news/computer-coding-computer-program-definition-and-code-meaning/"
article13title: "What is Coding?"
article14link: "https://www.freecodecamp.org/news/text-box-in-html-the-input-field-html-tag/"
article14title: "Text Box in HTML"
article15link: "https://www.freecodecamp.org/news/meta-tag-in-html-what-is-metadata-and-meta-description-example/"
article15title: "Meta Tag in HTML"
article16link: "https://www.freecodecamp.org/news/append-in-python-how-to-append-to-a-list-or-an-array/"
article16title: "Append in Python"
article17link: "https://www.freecodecamp.org/news/python-not-equal-how-to-use-the-does-not-equal-operator/"
article17title: "Python Not Equal"
article18link: "https://www.freecodecamp.org/news/the-linux-awk-command-linux-and-unix-usage-syntax-examples/"
article18title: "Linux awk Command"
article19link: "https://www.freecodecamp.org/news/javascript-string-contains-how-to-use-js-includes/"
article19title: "JS String Contains"
article1link: "https://www.freecodecamp.org/news/learn-java-free-java-courses-for-beginners/"
article1title: "Learn Java Courses"
article20link: "https://www.freecodecamp.org/news/how-to-open-task-manager-in-windows-10/"
article20title: "How to Open Task Manager"
article21link: "https://www.freecodecamp.org/news/the-design-thinking-process-explained/"
article21title: "Design Thinking Explained"
article22link: "https://www.freecodecamp.org/news/learn-programming-free-software-development-courses-for-beginners/"
article22title: "Learn Programming Courses"
article23link: "https://www.freecodecamp.org/news/transparent-taskbar-how-to-make-a-task-bar-transparent-in-windows-10-pc-2/"
article23title: "Make a Transparent Taskbar"
article24link: "https://www.freecodecamp.org/news/windows-10-network-adapter-missing-how-to-install-ethernet-driver-on-pc/"
article24title: "Install Ethernet Driver PC"
article25link: "https://www.freecodecamp.org/news/default-constructor-in-java/"
article25title: "Default Constructor in Java"
article26link: "https://www.freecodecamp.org/news/scanning-and-repairing-drive-how-to-fix-stuck-windows-10-pc-hard-drive/"
article26title: "Stuck Win 10 Hard Drive"
article27link: "https://www.freecodecamp.org/news/color-shades-hex-code-and-rgb-for-gray-color-palette/"
article27title: "Color Codes for Grey Palette"
article28link: "https://www.freecodecamp.org/news/binary-search-tree-traversal-inorder-preorder-post-order-for-bst/"
article28title: "Binary Search Tree Traversal"
article29link: "https://www.freecodecamp.org/news/rtc-connecting-discord-how-to-fix-the-server-error/"
article29title: "RTC Connecting Discord Fix"
article2link: "https://www.freecodecamp.org/news/sql-like-statement-how-to-query-sql-with-wildcard/"
article2title: "SQL Like Statement"
article3link: "https://www.freecodecamp.org/news/file-explorer-not-responding-fixed-on-windows-10-pc/"
article3title: "File Explorer Error"
article4link: "https://www.freecodecamp.org/news/python-find-in-list-how-to-find-the-index-of-an-item-or-element-in-a-list/"
article4title: "Python Find in List"
article5link: "https://www.freecodecamp.org/news/python-functions-define-and-call-a-function/"
article5title: "Functions in Python"
article6link: "https://www.freecodecamp.org/news/python-reverse-list-how-to-reverse-a-range-or-array/"
article6title: "Python Reverse List"
article7link: "https://www.freecodecamp.org/news/how-to-create-a-table-in-sql-postgres-and-mysql-example-query/"
article7title: "Create a Table in SQL"
article8link: "https://www.freecodecamp.org/news/list-index-out-of-range-python-error-message-solved/"
article8title: "List Index Out of Range"
article9link: "https://www.freecodecamp.org/news/python-string-replace-function-in-python-for-substring-substitution/"
article9title: "Python String.Replace()"
  }
```

Comment (ahmadabdolsaheb) [inline build/universal/trending/english.yaml:62]:
```suggestion
article0link: "https://www.freecodecamp.org/news/discord-wont-open-on-my-pc-solved-in-windows-10/"
article0title: "Discord Won't Open"
article10link: "https://www.freecodecamp.org/news/c-vs-cpp-whats-the-difference/"
article10title: "C vs C++"
article11link: "https://www.freecodecamp.org/news/python-json-how-to-convert-a-string-to-json/"
article11title: "Python JSON"
article12link: "https://www.freecodecamp.org/news/what-is-a-pc-computer-definition-and-computer-basics-for-beginners/"
article12title: "What is a PC?"
article13link: "https://www.freecodecamp.org/news/computer-coding-computer-program-definition-and-code-meaning/"
article13title: "What is Coding?"
article14link: "https://www.freecodecamp.org/news/text-box-in-html-the-input-field-html-tag/"
article14title: "Text Box in HTML"
article15link: "https://www.freecodecamp.org/news/meta-tag-in-html-what-is-metadata-and-meta-description-example/"
article15title: "Meta Tag in HTML"
article16link: "https://www.freecodecamp.org/news/append-in-python-how-to-append-to-a-list-or-an-array/"
article16title: "Append in Python"
article17link: "https://www.freecodecamp.org/news/python-not-equal-how-to-use-the-does-not-equal-operator/"
article17title: "Python Not Equal"
article18link: "https://www.freecodecamp.org/news/the-linux-awk-command-linux-and-unix-usage-syntax-examples/"
article18title: "Linux awk Command"
article19link: "https://www.freecodecamp.org/news/javascript-string-contains-how-to-use-js-includes/"
article19title: "JS String Contains"
article1link: "https://www.freecodecamp.org/news/learn-java-free-java-courses-for-beginners/"
article1title: "Learn Java Courses"
article20link: "https://www.freecodecamp.org/news/how-to-open-task-manager-in-windows-10/"
article20title: "How to Open Task Manager"
article21link: "https://www.freecodecamp.org/news/the-design-thinking-process-explained/"
article21title: "Design Thinking Explained"
article22link: "https://www.freecodecamp.org/news/learn-programming-free-software-development-courses-for-beginners/"
article22title: "Learn Programming Courses"
article23link: "https://www.freecodecamp.org/news/transparent-taskbar-how-to-make-a-task-bar-transparent-in-windows-10-pc-2/"
article23title: "Make a Transparent Taskbar"
article24link: "https://www.freecodecamp.org/news/windows-10-network-adapter-missing-how-to-install-ethernet-driver-on-pc/"
article24title: "Install Ethernet Driver PC"
article25link: "https://www.freecodecamp.org/news/default-constructor-in-java/"
article25title: "Default Constructor in Java"
article26link: "https://www.freecodecamp.org/news/scanning-and-repairing-drive-how-to-fix-stuck-windows-10-pc-hard-drive/"
article26title: "Stuck Win 10 Hard Drive"
article27link: "https://www.freecodecamp.org/news/color-shades-hex-code-and-rgb-for-gray-color-palette/"
article27title: "Color Codes for Grey Palette"
article28link: "https://www.freecodecamp.org/news/binary-search-tree-traversal-inorder-preorder-post-order-for-bst/"
article28title: "Binary Search Tree Traversal"
article29link: "https://www.freecodecamp.org/news/rtc-connecting-discord-how-to-fix-the-server-error/"
article29title: "RTC Connecting Discord Fix"
article2link: "https://www.freecodecamp.org/news/sql-like-statement-how-to-query-sql-with-wildcard/"
article2title: "SQL Like Statement"
article3link: "https://www.freecodecamp.org/news/file-explorer-not-responding-fixed-on-windows-10-pc/"
article3title: "File Explorer Error"
article4link: "https://www.freecodecamp.org/news/python-find-in-list-how-to-find-the-index-of-an-item-or-element-in-a-list/"
article4title: "Python Find in List"
article5link: "https://www.freecodecamp.org/news/python-functions-define-and-call-a-function/"
article5title: "Functions in Python"
article6link: "https://www.freecodecamp.org/news/python-reverse-list-how-to-reverse-a-range-or-array/"
article6title: "Python Reverse List"
article7link: "https://www.freecodecamp.org/news/how-to-create-a-table-in-sql-postgres-and-mysql-example-query/"
article7title: "Create a Table in SQL"
article8link: "https://www.freecodecamp.org/news/list-index-out-of-range-python-error-message-solved/"
article8title: "List Index Out of Range"
article9link: "https://www.freecodecamp.org/news/python-string-replace-function-in-python-for-substring-substitution/"
article9title: "Python String.Replace()"
```

Comment (ahmadabdolsaheb) [inline build/universal/trending/english.yaml:62]:
```suggestion
article0title: "Discord Won't Open"
article0link: "https://www.freecodecamp.org/news/discord-wont-open-on-my-pc-solved-in-windows-10/"
article1title: "Learn Java Courses"
article1link: "https://www.freecodecamp.org/news/learn-java-free-java-courses-for-beginners/"
article2title: "SQL Like Statement"
article2link: "https://www.freecodecamp.org/news/sql-like-statement-how-to-query-sql-with-wildcard/"
article3title: "File Explorer Error"
article3link: "https://www.freecodecamp.org/news/file-explorer-not-responding-fixed-on-windows-10-pc/"
article4title: "Python Find in List"
article4link: "https://www.freecodecamp.org/news/python-find-in-list-how-to-find-the-index-of-an-item-or-element-in-a-list/"
article5title: "Functions in Python"
article5link: "https://www.freecodecamp.org/news/python-functions-define-and-call-a-function/"
article6title: "Python Reverse List"
article6link: "https://www.freecodecamp.org/news/python-reverse-list-how-to-reverse-a-range-or-array/"
article7title: "Create a Table in SQL"
article7link: "https://www.freecodecamp.org/news/how-to-create-a-table-in-sql-postgres-and-mysql-example-query/"
article8title: "List Index Out of Range"
article8link: "https://www.freecodecamp.org/news/list-index-out-of-range-python-error-message-solved/"
article9title: "Python String.Replace()"
article9link: "https://www.freecodecamp.org/news/python-string-replace-function-in-python-for-substring-substitution/"
article10title: "C vs C++"
article10link: "https://www.freecodecamp.org/news/c-vs-cpp-whats-the-difference/"
article11title: "Python JSON"
article11link: "https://www.freecodecamp.org/news/python-json-how-to-convert-a-string-to-json/"
article12title: "What is a PC?"
article12link: "https://www.freecodecamp.org/news/what-is-a-pc-computer-definition-and-computer-basics-for-beginners/"
article13title: "What is Coding?"
article13link: "https://www.freecodecamp.org/news/computer-coding-computer-program-definition-and-code-meaning/"
article14title: "Text Box in HTML"
article14link: "https://www.freecodecamp.org/news/text-box-in-html-the-input-field-html-tag/"
article15title: "Meta Tag in HTML"
article15link: "https://www.freecodecamp.org/news/meta-tag-in-html-what-is-metadata-and-meta-description-example/"
article16title: "Append in Python"
article16link: "https://www.freecodecamp.org/news/append-in-python-how-to-append-to-a-list-or-an-array/"
article17title: "Python Not Equal"
article17link: "https://www.freecodecamp.org/news/python-not-equal-how-to-use-the-does-not-equal-operator/"
article18title: "Linux awk Command"
article18link: "https://www.freecodecamp.org/news/the-linux-awk-command-linux-and-unix-usage-syntax-examples/"
article19title: "JS String Contains"
article19link: "https://www.freecodecamp.org/news/javascript-string-contains-how-to-use-js-includes/"
article20title: "How to Open Task Manager"
article20link: "https://www.freecodecamp.org/news/how-to-open-task-manager-in-windows-10/"
article21title: "Design Thinking Explained"
article21link: "https://www.freecodecamp.org/news/the-design-thinking-process-explained/"
article22title: "Learn Programming Courses"
article22link: "https://www.freecodecamp.org/news/learn-programming-free-software-development-courses-for-beginners/"
article23title: "Make a Transparent Taskbar"
article23link: "https://www.freecodecamp.org/news/transparent-taskbar-how-to-make-a-task-bar-transparent-in-windows-10-pc-2/"
article24title: "Install Ethernet Driver PC"
article24link: "https://www.freecodecamp.org/news/windows-10-network-adapter-missing-how-to-install-ethernet-driver-on-pc/"
article25title: "Default Constructor in Java"
article25link: "https://www.freecodecamp.org/news/default-constructor-in-java/"
article26title: "Stuck Win 10 Hard Drive"
article26link: "https://www.freecodecamp.org/news/scanning-and-repairing-drive-how-to-fix-stuck-windows-10-pc-hard-drive/"
article27title: "Color Codes for Grey Palette"
article27link: "https://www.freecodecamp.org/news/color-shades-hex-code-and-rgb-for-gray-color-palette/"
article28title: "Binary Search Tree Traversal"
article28link: "https://www.freecodecamp.org/news/binary-search-tree-traversal-inorder-preorder-post-order-for-bst/"
article29title: "RTC Connecting Discord Fix"
article29link: "https://www.freecodecamp.org/news/rtc-connecting-discord-how-to-fix-the-server-error/"
```

**Comment by ahmadabdolsaheb** [comment]
The lengths have been tested. We could get this month's footers in and start working on automating it so we have a working solution for next month.

**Comment by ieahleen** [comment]
@ahmadabdolsaheb seems a good idea, I will set up a When2Meet for this

**Pull request merged**